### PR TITLE
Update for docsmobile v2.0.0

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Updates content for [Docsmobile v2.0.0](https://github.com/elastic/docsmobile/blob/main/docsmobile/CHANGELOG.md#200).

### CODE REVIEW

File: serverless/pages/sign-up.mdx [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 tags: [ 'serverless', 'general', 'signup' ]
 ---
 
 There are two options to create serverless projects:
 
-- If you are an existing customer, [log in to Elastic Cloud](https://cloud.elastic.co/login). On the home page, you will see a new option to create serverless projects. Note that if you are already subscribed to Elastic Cloud, there is no specific trial for serverless projects. 
+- If you are an existing customer, [log in to Elastic Cloud](https://cloud.elastic.co/login). On the home page, you will see a new option to create serverless projects. Note that if you are already subscribed to Elastic Cloud, there is no specific trial for serverless projects.
 
-- If you are a new user, you can [sign up for a free 14-day trial](https://cloud.elastic.co/serverless-registration), and you will be able to launch a serverless project.
+- If you are a new user, you can [sign up for a free 14-day trial](https://cloud.elastic.co/registration), and you will be able to launch a serverless project.
```
Comment (colleenmcginnis) [inline serverless/pages/sign-up.mdx:12]:
```suggestion
- If you are a new user, you can [sign up for a free 14-day trial](https://cloud.elastic.co/serverless-registration), and you will be able to launch a serverless project.
```

File: serverless/pages/manage-your-project-rest-api.mdx [hunk 2]
```diff
@@ -8,7 +7,7 @@ tags: [ 'serverless', 'project', 'manage', 'rest', 'api']
 
 <DocBadge template="technical preview" />
 
-You can manage serverless projects using the [Elastic Cloud Serverless REST API](https://www.elastic.co/docs/api/doc/elastic-cloud-serverless). This API allows you to create, update, and delete projects, as well as manage project features and usage.
+You can manage serverless projects using Elastic Cloud's [Project Management](https://docs.elastic.co/api-reference/project-management) REST API. This API allows you to create, update, and delete projects, as well as manage project features and usage.
```
Comment (colleenmcginnis) [inline serverless/pages/manage-your-project-rest-api.mdx:10]:
```suggestion
You can manage serverless projects using the [Elastic Cloud Serverless REST API](https://www.elastic.co/docs/api/doc/elastic-cloud-serverless). This API allows you to create, update, and delete projects, as well as manage project features and usage.
```

File: serverless/pages/manage-your-project-rest-api.mdx [hunk 3]
```diff
@@ -44,7 +43,7 @@ curl -H "Authorization: ApiKey essu_..." https://api.elastic-cloud.com/api/v1/se
 
 The Project Management API is documented using the [OpenAPI Specification](https://en.wikipedia.org/wiki/OpenAPI_Specification). The current supported version of the specification is `3.0`.
 
-For details, check the [API reference](https://www.elastic.co/docs/api/doc/elastic-cloud-serverless) or download the [OpenAPI Specification](https://www.elastic.co/docs/api/doc/elastic-cloud-serverless.yaml).
+For details, check the [API reference](https://docs.elastic.co/api-reference) or download the [OpenAPI Specification](https://docs.elastic.co/api-reference/openapi/serverless-project-api.yml).
```
Comment (colleenmcginnis) [inline serverless/pages/manage-your-project-rest-api.mdx:46]:
```suggestion
For details, check the [API reference](https://www.elastic.co/docs/api/doc/elastic-cloud-serverless) or download the [OpenAPI Specification](https://www.elastic.co/docs/api/doc/elastic-cloud-serverless.yaml).
```

**Review by colleenmcginnis — COMMENTED** [review]
Found a couple unexpected edits.

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-4a6zbvtrv-elastic-dev.vercel.app

* Built with commit bcba7abcfe1bbf76d801216992a81b180ba4d099

> Issues? Visit #next-docs in Slack

**Pull request merged**